tools/b43-tools/Makefile

```diff
@@ -13,8 +13,8 @@ PKG_VERSION:=019
 PKG_SOURCE_URL:=https://github.com/mbuesch/b43-tools.git
 PKG_SOURCE_PROTO:=git
 PKG_SOURCE_SUBDIR:=$(PKG_NAME)
-PKG_SOURCE_VERSION:=8dce53297966b31b6c70a7a03c2433978dd9f288
-PKG_MIRROR_HASH:=247eb0297f14fbcc828685cc353150f30fa03bf886e7a477aadc4161d9c0458d
+PKG_SOURCE_VERSION:=27892ef741e7f1d08cb939744f8b8f5dac7b04ae
```

Comment:
The PKG_VERSION should also be cahnged.
Either we change the version number to 
PKG_VERSION:=019-1
or we change to 
PKG_DATE:=2017-09-13

Reply:
Done